tests/scripts/shared.ts

```diff
@@ -1,6 +1,6 @@
 import assert from 'node:assert';
 import fs from 'node:fs';
-import { dirname, join, normalize } from 'node:path';
+import path, { dirname, join, normalize } from 'node:path';
```

Comment:
why not use `{ basename }`

Reply:
I was intended to avoid the name conflict.